common/test/resources/karma-browserstack-config.cjs

```diff
@@ -4,9 +4,9 @@
 var PRIMARY_LAUNCHERS = {
   bs_safari_mac_m: {
     browser: 'safari',
-    browser_version: '15.3',
+    browser_version: '17.3',
     os: 'OS X',
-    os_version: 'Monterey'
+    os_version: 'Sonoma'
```

Comment:
It seems as though the version of Safari we'd been targeting for testing with BrowserStack didn't like the updated Chai assertion library.  Updating the target to something more recent bypasses the issue.

Our goal with the 🏃 PR series is migrate away from CI testing against BrowserStack anyway, so I consider this a "stopgap" measure we can use to keep the tests passing during the test-migration process.